filter_directories.py

```diff
@@ -2,6 +2,12 @@
 
 import os
 import sys
+import lxml.etree as etree
+
+def is_tool(path):
+    xml = etree.parse(path)
+    return xml.getroot().tag == 'tool'
```

Comment:
galaxy-lib and planemo have abstractions for this stuff - we should get this all in planemo at some point.